src/ocm/components/clusterConfiguration/staticIp/data/formDataFromInfraEnvField.ts

```diff
@@ -145,7 +153,8 @@ const getFormViewHost = (
     },
   };
   for (const protocolVersion of getShownProtocolVersions(protocolType)) {
-    ret.ips[protocolVersion] = getIpAddress(ethernetInterface, protocolVersion);
+    if (vlanInterface) ret.ips[protocolVersion] = getIpAddress(vlanInterface, protocolVersion);
+    else ret.ips[protocolVersion] = getIpAddress(ethernetInterface, protocolVersion);
```

Comment:
Perhaps this is just my personal preference here but could we add brackets and indentation here? Or use a ternary operator and have just one assignment? Something about this notation just feels cluttered and hard to read.

Reply:
@ammont82 
the if and else are in separate lines separated by a semicolon... 
does this work?

Reply:
suggestion:
const interface = !!vlanInterface ? vlanInterface : ethernetInterface;
ret.ips[protocolVersion] = interface

Reply:
I agree, I also personally prefer the style with brackets etc.

But I think that it shouldn't be part of PRs, rather we should discuss it, and agree to enforce it with Prettier and have this fixed automatically.

Reply:
@celdrake the ternary operation is not the issue I think, I agree that doesn't need to be part of the PR.
The problem is the if and else statements are separated by a semicolon.
It got me curious about how this behave:

When I ran this I got an error:
const a = 1;
if (a) {console.log("b")};
else {console.log("c")};

SyntaxError: Unexpected token 'else'

Reply:
I can change. Is a personal preference. @batzionb if you run "npm run test" in this branch all is ok. Maybe the problem with your sample is because you're using brackets.
I agree with Celia that this shouldn't be part of the PR, we can enforce Prettier to fixed this automatically.

Reply:
I propose to change this condition to:
   vlanInterface ? ret.ips[protocolVersion] = getIpAddress(vlanInterface, protocolVersion):ret.ips[protocolVersion] = getIpAddress(ethernetInterface, protocolVersion);d

Reply:
Does that work? Do you not need to start with `ret.ips[protocolVersion]` as the left side of the assignment and have the ternary operator on the right side?

Reply:
@ammont82 you're right! without the brackets it works :)
Never saw this syntax before

Reply:
@jgyselov I checked this syntax works

Reply:
@batzionb I had used it in Javascript: https://iqcode.com/code/javascript/javascript-if-without-brackets

Reply:
cool!

Reply:
@jgyselov it works